File: app/src/main/java/com/syd/MainActivity.java
package com.syd;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import com.syd.studykotlin.R;
import com.syd.ui.main.MainFragment;

import org.w3c.dom.Document;
import org.w3c.dom.Node;

import javax.xml.parsers.DocumentBuilder;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.main_activity);
        if (savedInstanceState == null) {
            getSupportFragmentManager().beginTransaction()
                    .replace(R.id.container, MainFragment.newInstance())
                    .commitNow();
        }

    }
}


class Person {
    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    private String name;
    public Person(String name){
        this.name = name;
    }

}

File: app/src/main/java/com/syd/studykotlin/MainActivity.kt
package com.syd.studykotlin

import android.support.v7.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }
}

fun main(args : Array<String>){
    println("hello kotlin !")
    var a = 33
    a = "2222"
}


internal class Person(var name: String?)
